app/revenue/models.py

```diff
@@ -19,13 +19,131 @@
 """
 from django.contrib.postgres.fields import JSONField
 from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
 
 from dashboard.models import SendCryptoAsset
+from economy.models import SuperModel
 
 
-class DigitalGoodPurchase(SendCryptoAsset):
-    """Model that represents a DigitalGoodPurchase.
+class ALaCartePurchase(SendCryptoAsset):
+    """Model that represents a ALaCartePurchase.
 
     """
     purchase = JSONField(default=dict, blank=True)
     purchase_expires = models.DateTimeField(null=True, blank=True)
+    sku = models.ForeignKey(
+        'revenue.SKU',
+        related_name='alacartegoodpurchases',
+        on_delete=models.CASCADE,
+        help_text=_('The feature that was purchased'),
+        null=False,
+    )
+
+
+class SKU(SuperModel):
```

Comment:
what does SKU mean ? @owocki

Reply:
Typically a SKU would represent a type of product (a specific car model, like a Red Honda Civic 2019 with sunroof, for example). I think in this case its representing a specific type of A La Carte item purchased (a codefund impression, or kudos, or job board posting).